cmd/cli/handler_client.go

```diff
@@ -148,3 +148,25 @@ func (h *ClientHandler) GetClient(cmd *cobra.Command, args []string) {
 	checkResponse(err, http.StatusOK, response)
 	fmt.Println(cmdx.FormatResponse(&cl))
 }
+
+func (h *ClientHandler) ListClient(cmd *cobra.Command, args []string) {
+	m := h.newClientManager(cmd)
+
+	limit := flagx.MustGetInt(cmd, "limit")
+	page := flagx.MustGetInt(cmd, "page")
+	offset := (limit * page) - limit
+
+	cls, response, err := m.ListOAuth2Clients(int64(limit), int64(offset))
+	checkResponse(err, http.StatusOK, response)
+
+	table := newTable()
+	table.SetHeader([]string{"Client ID", "Name", "Response Type", "Scope"})
```

Comment:
`Response Type` should be `Response Types`. I also think it would be important to add the following fields:

* `RedirectUris`
* `GrantTypes`
* `TokenEndpointAuthMethod`

Reply:
Exactly! Have added :)